src/main/java/lottogame/model/lotto/ManualLottos.java

```diff
@@ -0,0 +1,35 @@
+package lottogame.model.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class ManualLottos {
+    private final List<Set<Integer>> manualLottos;
```

Comment:
```suggestion
    private final List<Lotto> manualLottos;
```
Lottos와 다른 점이 있나?
굳이 분리할 필요가 있을까?